website/docs/docs/partnerships.mdx

```diff
@@ -154,15 +146,17 @@ HashiCorp Terraform has an extremely wide community of users and contributors, a
 
 Vendors who choose not to support their provider and prefer to make it a community-supported provider will no longer be listed as Partner.
 
+-> Terraform modules published by Partner: We now have the ability to tag partner published modules in the Terraform Registry. If you have a modules that you’d like to receive the “Partner” tag, please reach out to us at technologypartners@hashicorp.com
```

Comment:
```suggestion
-> Terraform modules published by Partner: We now have the ability to tag partner published modules in the Terraform Registry. If you have a module that you want to receive the “Partner” tag, please contact technologypartners@hashicorp.com
```

Suggest changing "you'd like" to "you want" to avoid contractions. Suggest changing "reach out to us" to "contact" to use fewer words.